src/index.js

```diff
@@ -2,6 +2,7 @@ import './style.css';
 import todoAdd from './modules/add';
 import todoRemove from './modules/remove';
 import todoEdit from './modules/edit';
```

Comment:
- [ ] [Optional] Yout all functionality is working as expected😃 , but I see that when I remove the todo separately not using the clear button, I see that when I remove 🚀the todo and reload the page there is an error with the remove functionality from the previous milestone, so it will be better if you fix that to make your todo complete for all functionality 💯😊

- Hint: 
- Check the video [here](https://www.loom.com/share/4a4e2e3bbafc4f3ab05f9d4fbb0a6c64) 😊⤵️for your project when I try to remove the todo you will notice that the clear functionality is working, but the remove todo from the previous milestone gave me an error, so it will be better if you fix that to make your todo complete for all functionality 💯😊

Reply:
Yes Brother i am unable to find the root cause of this buggy behaiour; but i will try to resolve it.